compass/landice/mesh.py

```diff
@@ -167,6 +167,18 @@ def set_cell_width(self, section, thk, vx=None, vy=None,
         spacing = np.interp(lspd, [low_log_speed, high_log_speed],
                             [max_spac, min_spac], left=max_spac,
                             right=min_spac)
+
+        # Clean up where we have missing velocities. These are usually nans
+        # or the default netCDF _FillValue of ~10.e36
+        missing_data_mask = np.logical_or(
+                               np.logical_or(np.isnan(vx), np.isnan(vy)),
+                               np.logical_or(np.abs(vx) > 1.e5,
+                                             np.abs(vy) > 1.e5))
+        spacing[missing_data_mask] = max_spac
+        logger.info('Found {} points in input dataset with missing ' \
+                     'velocity values. Setting velocity-based spacing ' \
+                     'to maximum value.'.format(np.sum(missing_data_mask)))
```

Comment:
```suggestion
        logger.info('Found {} points in input dataset with missing '
                    'velocity values. Setting velocity-based spacing '
                    'to maximum value.'.format(np.sum(missing_data_mask)))
```
These are from https://github.com/MPAS-Dev/compass/pull/271#issuecomment-965526634

You don't need a backslash for continuing lines that are already in parentheses like here.